app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -82,3 +65,4 @@
 <%= render 'shared/cancel', link: destroy_user_session_path %>
 
 <%= javascript_packs_tag_once('webauthn-unhide') %>
+<%= javascript_packs_tag_once('mfa_selection_component') %>
```

Comment:
Since the script applies to partial content, probably makes sense for the `javascript_packs_tag_once` call to occur inside the partial.